### ISSUE TITLE
[TF 2.0 API Docs] tf.nn.dropout

### ISSUE THREAD

**Issue by MagsMagnoli** [tensorflow/tensorflow#29277]
## URL(s) with the issue:

https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/nn/dropout

### Usage example

Usages are linked but none are detailed inline on the page

**Comment by amahendrakar** [comment]
@MagsMagnoli,
The documentation has been updated for TensorFlow v2.4. Could you please take a look at it and let us know if this is still an issue? Thanks!
